papaya-core/src/main/java/pl/edu/agh/papaya/service/sprint/SprintCreationWizard.java

```diff
@@ -9,8 +9,13 @@
 @SuppressWarnings({"checkstyle:HiddenField", "PMD.BeanMembersShouldSerialize"})
 public class SprintCreationWizard {
 
+    private static final String DEFAULT_NAME = "";
+    private static final String DEFAULT_NOTIFICATION_MESSAGE = "";
```

Comment:
Czyli jak ich nie ma, to są `""`,  a nie nullami? Nie wiem, czy dla mnie nulle nie byłyby "ładniejsze"

Reply:
jak od początku założymy że nazwa i message zawsze są to potem będzie łatwiej zmienić

Reply:
Znaczy _docelowo_ i _potem_ chyba dzieje się teraz

Reply:
W sensie to się sprowadza do tego, że ifujesz po tym, czy string jest pusty, i nie wiem, w czym to jest lepsze od nulli. Message raczej na bank nie jest wymagane, nazwa chyba też nie jak dla mnie